aep/general/0231/aep.md.j2

```diff
@@ -9,25 +9,40 @@ provides this functionality.
 APIs **may** support batch get to retrieve a consistent set of resources.
 
 - The method's name **must** begin with `BatchGet`. The remainder of the method
-  name **should** be the plural form of the resource being retrieved.
+  name **must** be the plural form of the resource being retrieved.
 - The HTTP verb **must** be `GET`.
-- The HTTP URI **must** end with `:batchGet`.
-- The URI path **should** represent the collection for the resource, matching
-  the collection used for simple CRUD operations. If the operation spans
-  parents, a dash (`-`) **may** be accepted as a wildcard.
+- The HTTP URI **must** end with `:BatchGet`.
+- The URI path **must** represent the collection for the resource, matching the
+  collection used for simple CRUD operations. If the operation spans parents, a
+  [wilcard](./reading-across-collections) **may** be accepted.
 - There **must not** be a request body.
-  - If a GET request contains a body, the body **must** be ignored, and **must not** cause an error.
-- The operation **must** be atomic: it **must** fail for all resources or
-  succeed for all resources (no partial success). For situations requiring
-  partial failures, `List` (AEP-132) methods **should** be used.
-  - If the operation covers multiple locations and at least one location is
-    down, the operation **must** fail.
+  - If a GET request contains a body, the body **must** be ignored, and **must
+    not** cause an error.
 
 ### Request
 
 The request for a batch get method **should** be specified with the following
 pattern:
 
+- The request **must** include an array parameter which accepts the resource
+  paths specifying the resources to retrieve. The parameter **should** be named
+  `paths`.
+  - If no resource paths are provided, the API **should** error with
+    `INVALID_ARGUMENT`.
+  - The parameter **should** be required.
+  - The parameter **should** identify the [resource type](./paths) that it
+    references.
+  - The parameter should define the pattern of the resource path values.
+  - The parameter should define the maximum number of paths allowed.
```

Comment:
Are these three bullet points all about documentation?

Reply:
In some ways all of this is about documentation, right? But in OpenAPI / JSON Schema many of these things can be described formally, such as with a `maxItems` keyword in an array schema.

Reply:
could we just update it to specify documentation? we could even qualify it in the way you described - if the protocol / description supports a way to describe it, use that. Otherwise use documentation.